sdk/keyvault/azure-keyvault-keys/tests/test_samples_keys_async.py

```diff
@@ -46,7 +46,7 @@ class TestExamplesKeyVault(KeyVaultTestCase):
     @recorded_by_proxy_async
     async def test_example_key_crud_operations(self, key_client, **kwargs):
         if (self.is_live and os.environ["KEYVAULT_SKU"] != "premium"):
-            pytest.skip("This test not supprot in usgov/china region. Follow up with service team")
+            pytest.skip("This test is not supported on standard SKU vaults. Follow up with service team")
```

Comment:
How about keeping some messages before the change?
```suggestion
            pytest.skip("This test is not supported in usgov/china region. Follow up with service team")
```